assets/src/edit-story/utils/getMediaBaseColor.js

```diff
@@ -65,7 +62,12 @@ function getDefaultOnloadCallback(nodeKey, resolve, reject) {
   return () => {
     try {
       const node = document.body[nodeKey];
-      resolve(thief.getColor(node.firstElementChild));
+      import(
+        /* webpackPrefetch: true, webpackChunkName: "chunk-colorthief" */ 'colorthief'
```

Comment:
Any idea why the file for this one is *not* called `vendors-chunk-colorthief` like the other files, but just `chunk-colorthief`? @swissspidy?

Reply:
This is due to the size. The default threshold is 3KB for a separate chunk to be generated for loading separately.

Docs: https://v4.webpack.js.org/plugins/split-chunks-plugin/#optimizationsplitchunks